opencensus/common/transports/async.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atexit
+import threading
+import time
+
+from six.moves import queue
+from six.moves import range
+
+from opencensus.common.transports import base
+
+_DEFAULT_GRACE_PERIOD = 5.0  # Seconds
+_DEFAULT_MAX_BATCH_SIZE = 10
+_WAIT_PERIOD = 1.0  # Seconds
+_WORKER_THREAD_NAME = 'opencensus.common.Worker'
+_WORKER_TERMINATOR = object()
+
+
+class _Worker(object):
+	"""A background thread that exports batches of data.
```

Comment:
The indentation of this file is different than other files in this repo, shall we change to use 4 spaces indentation to match with the repo?